test/scripts/proxysql_stop.sh

```
@@ -0,0 +1,33 @@
+	killall mysql 2>/dev/null || true
+	sleep 5
+	killall -9 proxysql 2>/dev/null || true
+	killall -9 mysql 2>/dev/null || true
+}
+
+SECONDS=0
+echo "[$(date '+%Y-%m-%d %H:%M:%S')] >>> Stopping ProxySQL ..."
+
+(mysql -h127.0.0.1 -P6032 -uadmin -padmin -e"PROXYSQL SHUTDOWN SLOW" 2>&1 || true ) &
+
+local counter=0
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)